securedrop_client/locale/messages.pot

```diff
@@ -313,3 +310,6 @@ msgstr ""
 msgid "Files and messages"
 msgstr ""
 
+msgid "USE OFFLINE"
+msgstr ""
+
```

Comment:
These `.pot` changes are required due to the refactor. During review, @gonzalo-bulnes mentioned that CI was smart enough to fail here pending regeneration of this file. So hat-tip to @cfm on the CI logic, making sure this file is up to date!